guides/operator/orca-redis-to-sql.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: single
+title:  "Orca: Redis to SQL"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Orca's execution state is stored in Redis by default, but can be configured for SQL. In this topology, Redis will still be required for the work queue.
```

Comment:
I would think this should live in "setup" rather than "guides", but open to either. cc @dorbin

Reply:
@lwander and I just had a talk about this, and his idea is to split this up into 
* Set up Orca to use SQL (in /setup/)
* Migrate existing Orca config to use SQL (here in /guides/operator/)

WDYT, @robzienert ?

Reply:
I can get behind that!